packages/wonder-blocks-badge/src/components/badge.tsx

```diff
@@ -66,9 +70,14 @@ export {Badge};
 const badgeTokens = {
     root: {
         layout: {
-            paddingBlock: sizing.size_040,
-            paddingInline: sizing.size_080,
-            gap: sizing.size_040,
+            default: {
+                paddingBlock: sizing.size_040,
+                paddingInline: sizing.size_080,
+                gap: sizing.size_040,
+            },
+            iconOnly: {
```

Comment:
moved `iconOnly` styles to under `root` since it applies to the root element of the component (iconOnly is more of a modifier for `root`)